src/components/InventoryItem/InventoryItem.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+export default function InventoryItem({item}) {
+  const name = Object.keys(item)[0].replace('_', ' ')
+  const {description,weight,type} = Object.values(item)[0];
+  return (
+    <tr>
```

Comment:
Try not to use the tag table for style, just when you are rendering data as a table